indexer/services/socks/__tests__/lib/message-forwarder.test.ts

```
@@ -174,10 +174,10 @@ describe('message-forwarder', () => {
     childSubaccount2Messages[1],
   ];
 
-  const mockAxiosResponse: Object = { a: 'b' };
+  const mockAxiosResponse: Object[] = ['a', 'b'];
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)